advanced-client-install.rst

```diff
@@ -33,12 +33,12 @@ Windows will by default have firewall rules that prevent inbound connections. If
 
 ``netsh advfirewall firewall add rule name="Allow from <peer private addr>" dir=in action=allow protocol=ANY remoteip=<peer private addr>``
 
-If you want to allow all peers access, but do not want to configure firewall rules for all peers, you can configure access for one peer, and set it as a Relay Server.
+If you want to allow all peers access, but do not want to configure firewall rules for all peers, you can configure access for one peer, and set it as a Relay Server(Enterprise Edition Feature) from Netmaker GUI. In order to achieve this, self-hosted netmaker enterprise edition server is required.
```

Comment:
It doesn't have to be self-hosted, a pro version can be created on SaaS as well
Change `Enterprise` terminology to `Professional`

Reply:
Pushed a commit with amendments.